g1ant-language/triggering/README.md

```diff
@@ -18,20 +20,24 @@ Note that some possible arguments are not required to be set because they alread
 </Trigger> 
 ```
 
-**Arguments to define a trigger:**
-`Class` - describes what type of trigger it is (possible: FileTrigger, ScheduleTrigger, MailTrigger)
-`Name` - unique name declared to distinguish triggers from the same Class
-`TaskName` - either a name of a robot script without extension that we want to launch, name of it with extension or just a path to this robot script
+### Arguments to Define a Trigger
+
+| Argument   | Description                                                  |
+| ---------- | ------------------------------------------------------------ |
+| `Class`    | Describes the type of a trigger: `FileTrigger`, `ScheduleTrigger` or `MailTrigger` |
+| `Name`     | A unique name to distinguish triggers from the same `Class`  |
+| `TaskName` | Either a name of a robot script that we want to launch (with or without the file extension) or a full path to this robot script |
+
+### An Argument Defining Trigger Initial Arguments
 
-**Argument to define trigger initial arguments:**
-`Key` - name of a trigger initial argument, value of that trigger argument should be put between `<Argument>` ... `</Argument>`.
+`Key` is the name of a trigger initial argument. The value of that trigger argument should be put between `<Argument>` ... `</Argument>`.
 
-**Task Arguments**
+## Task Arguments
 
-Arguments generated while executing the script. In order to get them, use special variable `♥task` and inside these special rectangle characters `⟦⟧` insert the name of an argument.
-e.g. `♥task⟦filepath⟧`
+Task arguments are generated while executing the script. In order to get them, use a special variable `♥task` and insert the name of an argument inside special double brackets `⟦⟧`,
+e.g. `♥task⟦filepath⟧`.
 
-For the list of possible task or initial arguments, please visit manual page regarding a specific trigger.
+For the list of possible task or initial arguments, see the description of a specific trigger.
 
-Note that triggers are not active at the start of G1ANT.Robot. Whenever we want to use them, the triggersactive option must be switched on before launching the script. You can do this by choosing `Triggers/Active` from the Menu in Robot.
+Note that triggers are not active when G1ANT.Robot starts. Whenever you want to use them, you have to activate them before launching the script. You can do this by choosing `Triggers/Active` menu in Robot window.
```

Comment:
in developer version triggers are not active when G1ANT.Robot starts but in production version they are active - you need to update this info

it's not true that "Whenever you want to use them, you have to activate them before launching the script." once you activate them, they will be active until you close G1ANT.Robot

Reply:
OK, your remarks included.